models/boards.js

```diff
@@ -10,6 +10,66 @@ Boards.attachSchema(new SimpleSchema({
      */
     type: String,
   },
+  boardKey: {
+    /**
+     * The key of the board
+     */
+    type: String,
+    min: 1,
+    max: 3,
```

Comment:
This should be larger than 3 maximum. It should be a configurable setting in the admin section. The `max` key can take a function that returns what the maximum is. You can use this to lookup the admin setting for maximum. Make the admin maximum setting readable by all users but only writeable by admins.